recipes/qsv/build.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -o xtrace -o nounset -o pipefail -o errexit
+
+cargo-bundle-licenses \
+    --format yaml \
+    --output THIRDPARTY.yml
+
+# build statically linked binary with Rust
+export LIBCLANG_PATH="${PREFIX}/lib"
+if [[ ${target_platform} =~ .*osx.* ]]; then
+    export CFLAGS="${CFLAGS} -fno-define-target-os-macros"
+fi
+
+# If we are NOT cross compiling, supply the path to the python interpreter given by ${PYTHON}
+# If we are cross compiling, the cross-python package will provide these variables instead.
+if [[ ${build_platform} == ${target_platform} ]]; then
+    export PYO3_PYTHON=${PYTHON}
+fi
+
+cargo install --features all_features --locked --root ${PREFIX} --path .
```

Comment:
you can add `--no-track` here and remove the `rm -rf .crates.toml` at the end.

Reply:
Thanks, I made this change and will keep that in mind for future recipes.